pretty.js

```diff
@@ -56,6 +56,8 @@ function pretty (opts) {
   var timeTransOnly = opts && opts.timeTransOnly
   var formatter = opts && opts.formatter
   var levelFirst = opts && opts.levelFirst
+  var messageProp = opts && opts.messageProp
```

Comment:
Added a `-m <key>` option for the CLI tool. It's a bit heavier than the `-t` and `-l` flags but hopefully simple enough to use

It doesn't support CLI arg formats like `-m<key>` or `-m=<key>` but without adding some heavy arg-parsing dependencies I think that's the best solution.